advocacy_docs/edb-postgres-ai/analytics/how_to_lakehouse_sync.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Lakehouse Sync
+navTitle: Lakehouse Sync
+description: How-to guide for performing a Lakehouse Sync.
+---
+
+## Overview
+
+A Lakehouse Sync is a way to capture information from a transactional database at a point in time and sync that information to a Managed Store Location in an AWS region.
+
+Once synced, the data in the MSL is organized into Lakehouse tables. These Lakehouse tables can then be queried by a Lakehouse node optimized for higher-performance queries using a vectorized query engine for Lakehouse tables.
+
+## Perfroming a Lakehouse Sync
+
+### Prerequisites
+
+- a Postgres cluster running on your EDB Postgres AI Cloud Service account
+
+### Navigate to Lakehouse Sync
+
+1. Login to your EDB Postgres AI Console.
+
+2. From the landing page, select the project with the database instance you want to sync. If it is not shown on the landing page, select the **View Projects** link in the **Projects** section and select your project from there.
```

Comment:
Yes it is just the landing page, or at least that's what I remember Jace saying.

Reply:
Yes I personally think it's easier to tell people to "go to the cluster page for the transactional cluster you want to sync, click 'quick actions,' and 'Sync to Lakehouse' (this is what DJ shows in his screenshot)."

The Project -> Migrations -> Create Migration path is actually not the ideal one, especially because when you start a migration and need to select from a number of sources, it's not super obvious from the dropdown which source is which...